atlite/resource.py

```diff
@@ -180,3 +184,172 @@ def smooth(velocities, power):
+        # Only consider turbines with power curves available
+        df = pd.DataFrame.from_dict(result.json())
+        _oedb_turbines = df[df.has_power_curve]
+
+
+    logger.info("Searching turbine power curve in OEDB database using " +
+                ", ".join(f"{k}='{v}'" for (k,v) in turbine.items()) + ".")
+    # Working copy
+    df = _oedb_turbines
+    if turbine.get('id'):
+        df = df[df.id == int(turbine['id'])]
+    if turbine.get('name'):
+        df = df[df.turbine_type.str.contains(turbine['name'], case=False)]
+    if turbine.get('manufacturer'):
+        df = df[df.manufacturer.str.contains(turbine['manufacturer'], case=False)]
+
+
+    if len(df) < 1 :
+        logger.info("No turbine found.")
+        return None
+    elif len(df) > 1 :
+        logger.info(f"Provided information corresponds to {len(df)} turbines: \n"
+                    f"{df[['id','manufacturer','turbine_type']].head(3)}. \n"
+                    f"Use an 'id' for an unambiguous search.")
+        return None
+    elif len(df) == 1:
+        # Convert to series for simpliticty
+        ds = df.iloc[0]
+
+    # convert power from kW to MW
+    power = np.array(json.loads(ds.power_curve_values)) / 1e3
+
+    turbineconf = {
+        "name": ds.turbine_type.strip(),
+        "manufacturer": ds.manufacturer.strip(),
+        "source": f"Original: {ds.source}. Via OEDB {OEDB_URL}",
+        "HUB_HEIGHT": ds.hub_height,
+        "V": json.loads(ds.power_curve_wind_speeds),
+        "POW": power.tolist(),
+    }
+
+    # Other convinience assumptions
+    if not turbineconf['HUB_HEIGHT']:
+        turbineconf['HUB_HEIGHT'] = 100
+        logger.warning(f"No HUB_HEIGHT defined in dataset. Manual clean-up required."
+                       f"Assuming a HUB_HEIGHT of {turbineconf['HUB_HEIGHT']}m for now.")
+
+    if ";" in str(turbineconf['HUB_HEIGHT']):
+        hh = [float(t.strip()) for t in turbineconf['HUB_HEIGHT'].strip().split(";") if t]
+
+        if len(hh) > 1:
+            turbineconf['HUB_HEIGHTS'] = hh
+            turbineconf['HUB_HEIGHT'] = np.mean(hh, dtype=int)
+            logger.warning(f"Multiple HUB_HEIGHTS in dataset ({turbineconf['HUB_HEIGHTS']}). "
+                           f"Manual clean-up is required. "
+                           f"Using the average {turbineconf['HUB_HEIGHT']}m for now.")
+        else:
+            turbineconf['HUB_HEIGHT'] = hh[0]
+
+    if store_locally is True:
+        filename = (f"{turbineconf['manufacturer']}_{turbineconf['name']}.yaml"
+                     .replace('/','_').replace(' ','_').replace('-','_'))
+        filepath = construct_filepath(os.path.join(config.windturbine_dir, filename))
+
+        with open(filepath, 'w') as turbine_file:
+            yaml.dump(turbineconf, turbine_file)
+
+        _update_resource_dictionaries()
+        logger.info(f"Turbine configuration downloaded to '{filepath}'.")
+
+
+    return turbineconf
+
+# Global caches
+_oedb_turbines = None
+turbines = arrowdict()
```

Comment:
```suggestion
windturbines = arrowdict()
```
For consistency purposes.

Reply:
Food for thought on consistency:
```python
cutout.wind(turbine=atlite.windturbines.Vestas_V112_3MW)
cutout.pv(panel=atlite.solarpanels.CdTe)
```

Reply:
`wind(turbine` and `pv(panel` sound nice IMHO.

Reply:
Sold!